rust/kcl-wasm-lib/Cargo.toml

```diff
@@ -53,5 +53,5 @@ wasm-bindgen-test = "0.3.50"
 typed-path = "0.11.0"
 
 # Local development only. Placeholder to speed up development cycle
-#[package.metadata.wasm-pack.profile.release]
-#wasm-opt = false
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
```

Comment:
This needs to be reverted. I already commented on this, but I guess it got lost. Use `npm run build:wasm:dev` so that you don't need to change this file.